app/api/sessions/route.ts

```diff
@@ -1,7 +1,14 @@
 import { NextRequest, NextResponse } from 'next/server';
+import Stripe from 'stripe';
 import { stripeClient } from '../../../utils/stripe';
 
-export async function POST(request: NextRequest) {
+type StripeResponseBody = {
+  sessions: Stripe.Checkout.Session;
+};
+
+export async function POST(
+  request: NextRequest,
+): Promise<NextResponse<StripeResponseBody>> {
```

Comment:
Import the correct type from Stripe to use as the response body for the nest.js route handler